gradio/analytics.py

```diff
@@ -178,29 +178,34 @@ def launched_analytics(blocks: gradio.Blocks, data: dict[str, Any]) -> None:
         [],
     )
 
-    from gradio.blocks import BlockContext
-
     for x in list(blocks.blocks.values()):
-        blocks_telemetry.append(x.get_block_name()) if isinstance(
-            x, BlockContext
-        ) else blocks_telemetry.append(str(x))
-
+        blocks_telemetry.append(x.get_block_name())
```

Comment:
We were saving the string repr of the instance as opposed to the name, e.g. <gradio.components.textbox.Textbox object at 0x11c035790>